dbms/src/Flash/Statistics/ExecutorStatisticsCollector.h

```diff
@@ -86,6 +97,7 @@ class ExecutorStatisticsCollector
     std::map<String, ExecutorStatisticsPtr> profiles;
     const LoggerPtr log;
     bool force_fill_executor_id; // for testing list based executors
+    std::unique_ptr<resource_manager::Consumption> local_ru;
```

Comment:
```suggestion
    std::optional<resource_manager::Consumption> local_ru;
```
Maybe `optional` is better

Reply:
done